django/conf/locale/ar_DZ/formats.py

```diff
@@ -2,20 +2,30 @@
 #
 # The *_FORMAT strings use the Django date format syntax,
 # see https://docs.djangoproject.com/en/dev/ref/templates/builtins/#date
-DATE_FORMAT = 'j F Y'  # 04 نوفمبر 2020
-TIME_FORMAT = 'g:i A'
-# DATETIME_FORMAT =
+DATE_FORMAT = 'j F Y'  
+TIME_FORMAT = 'H:i'
+DATETIME_FORMAT = 'j F Y H:i'
 YEAR_MONTH_FORMAT = 'F Y'
 MONTH_DAY_FORMAT = 'j F'
-SHORT_DATE_FORMAT = 'd‏/m‏/Y'
-# SHORT_DATETIME_FORMAT =
+SHORT_DATE_FORMAT = 'Y/m/d'
```

Comment:
Are you sure? Should this not be consistent with `SHORT_DATETIME_FORMAT`, i.e. `SHORT_DATE_FORMAT = 'j N Y'`.

Reply:
Hi Felix,
You right for the SHORT_DATE_FORMAT

To summerize you, maybe you can help Felix, 

My application is a multilanguage app, mean needs a translations and formats, that's the reason why I needed a ar-DZ, but now I want to send that format used in django to a bootstrap datepicker, means that in our input case : DATE_INPUT_FORMATS : '%Y/%m/%d' I want to use it to convert it so that bootstrap datetime can use that format, like : 'yyyy/mm/dd', I know how to convert and use a snippet for that, but I didn't know where I can get that django date_input format that has been used to generate the field  'Y/m/d'. but the input in form stick to "d/m/Y' ??


Do you know how to get the current format used/applied to a date_field ? I wan't pass that to a bootstrap datepicker.

Here's a screenshot of a Update view well rendering date format in the ar-DZ :
<a>https://ibb.co/XCvC7rS</a>
and when you try to put a date into it using datebootstrap, wich is not well formated :
PS : I didn't define a date-format for bootstrap datepicker, I let it use the django default format country.
<a>https://ibb.co/7xfNY80</a>

Hope to hear from you soon.

Reply:
@infosrabah PR is not the best place to ask support question. Please use one of [ support channels](https://code.djangoproject.com/wiki/TicketClosingReasons/UseSupportChannels).